test/unit/instance-validator.test.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+/* jshint -W030 */
+const chai = require('chai');
+const expect = chai.expect;
+const Support = require(__dirname + '/support');
+const InstanceValidator = require('../../lib/instance-validator');
+const sinon = require('sinon');
+var Promise = Support.sequelize.Promise;
+const SequelizeValidationError = require('../../lib/errors').ValidationError;
+
+describe(Support.getTestDialectTeaser('InstanceValidator'), () => {
+
+  it('configures itself to run hooks by default', () => {
+    const instanceValidator = new InstanceValidator();
+    expect(instanceValidator.options.hooks).to.equal(true);
+  });
+
+  beforeEach(() => {
+    this.User = Support.sequelize.define('user');
+  });
+
+  describe('validate', () => {
+    it('runs the validation sequence and hooks when the hooks option is true', () => {
+      const instanceValidator = new InstanceValidator(this.User.build(), {hooks: true});
+      const _validate = sinon.spy(instanceValidator, '_validate');
+      const _validateAndRunHooks = sinon.spy(instanceValidator, '_validateAndRunHooks');
+      instanceValidator.validate();
```

Comment:
A new line before this line, please, and one after as well. Idea is to have something like this

```js
// define stuff

// do something

// expect things
```

Please improve test segments where this is applicable